src/SearchParameters/index.js

```diff
@@ -358,8 +363,8 @@ function SearchParameters(newParameters) {
     if (!this.hasOwnProperty(paramName)) {
       // IE8/9 has no console (BUT if devtools opened), nevermind there's no
       // developer working ONLY in IE8/9
-      window.console &&
-      window.console.error(
+      console &&
```

Comment:
Because it rely on window which breaks which unit testing with node. I will improve that.